exchanges/yobit/yobit_wrapper.go

```diff
@@ -235,12 +238,16 @@ func (y *Yobit) UpdateOrderbook(p currency.Pair, assetType asset.Item) (orderboo
 
 	for x := range orderbookNew.Bids {
 		data := orderbookNew.Bids[x]
```

Comment:
Out of scope: But you can use an index for these, rather than creating a new var